ext/dispatcher.go

```diff
@@ -143,7 +141,7 @@ func NewDispatcher(opts *DispatcherOpts) *Dispatcher {
 		errHandler = opts.Error
 		panicHandler = opts.Panic
 		unhandledErrFunc = opts.UnhandledErrFunc
-		errLog = opts.ErrorLog
+		logger = opts.Logger
```

Comment:
This is opinionated, but I find that having this type of helper function may help in some cases:
```go
func iftrue[T any](cond bool, t T, f T) T {
   if cond {
      return t
   }
   return f
}
```
(addressing the lack of  C-style ternary condition operator ?: in Go)

and then:
```go
logger = iftrue(opts.Logger == nil, logger, opts.Logger)
```

Reply:
Ooh, I like that, thank you!